smart_card_connector_app/src/create_human_readable_supported_readers.py

```diff
@@ -127,6 +127,9 @@ def main():
   args = args_parser.parse_args()
 
   readers_list = create_readers_list(args.ccid_supported_readers_file)
+  args.target_file.write("# NOTE: Auto-generated by "
+  "create_human_readable_supported_readers.py.\n# DO NOT MODIFY MANUALLY!\n\n")
```

Comment:
nit: Add spaces to align the opening `"` with the `"` in the previous line.

Or indent it like this:

```python
args.target_file.write(
    "..."
    "...")